src/frequenz/sdk/timeseries/formula_engine/_formula_generators/_battery_power_formula.py

```diff
@@ -64,7 +64,7 @@ def generate(
 
         component_graph = connection_manager.get().component_graph
 
-        battery_inverters = list(
+        battery_inverters = frozenset(
```

Comment:
was this supposed to go into a different commit?

Reply:
fixed >:|